config.go

```diff
@@ -146,7 +146,7 @@ type Config struct {
 	TLSExtraDomains []string `long:"tlsextradomain" description:"Adds an extra domain to the generated certificate"`
 	TLSAutoRefresh  bool     `long:"tlsautorefresh" description:"Re-generate TLS certificate and key if the IPs or domains are changed"`
 
-	NoMacaroons     bool          `long:"no-macaroons" description:"Disable macaroon authentication"`
+	NoMacaroons     bool          `long:"no-macaroons" description:"Disable macaroon authentication. This option is not allowed if the server is listening on a public interface."`
```

Comment:
nit: to follow the same pattern as most of the other descriptions that don't use full stops, this could be changed to `Disable macaroon authentication, can only be used if server is not listening on a public interface`

Reply:
got ya, fixed.